core/trino-main/src/main/java/io/trino/metadata/DisabledSystemSecurityMetadata.java

```diff
@@ -181,6 +181,15 @@ public void tableCreated(Session session, CatalogSchemaTableName table) {}
     @Override
     public void tableRenamed(Session session, CatalogSchemaTableName sourceTable, CatalogSchemaTableName targetTable) {}
 
+    @Override
+    public void columnCreated(Session session, CatalogSchemaTableName table, String column) {}
+
+    @Override
+    public void columnRenamed(Session session, CatalogSchemaTableName table, String oldName, String newName) {}
+
+    @Override
+    public void columnDropped(Session session, CatalogSchemaTableName table, String column) {}
+
```

Comment:
Move to after `tableDropped` method. Th current place is between `tableRenamed` and `tableDropped`.